jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -132,6 +144,33 @@
      (cons :action-type control_msgs::FollowJointTrajectoryAction)
      (cons :joint-names (list "left_gripper_vacuum_pad_joint")))))
 
+  ;; based on naoqi-interface and fetch-interface
+  (:move-hand
+    (arm av tm &key (wait t))
+    (if (and (= (length av) 2) (not (send self :simulation-modep)))
+      (if (eq arm :rarm)
+        (let (av-list (goal (instance control_msgs::FollowJointTrajectoryActionGoal :init))
+                      (traj (instance trajectory_msgs::JointTrajectory :init)))
+          (dotimes (i (length av)) (pushback (elt av i) av-list))
+          (send traj :header :seq 1)
+          (send traj :header :stamp (ros::time-now))
+          (send traj :joint_names
+                (list "right_gripper_finger_yaw_joint" "right_gripper_finger_roll_joint"))
+          (send traj :points
+                (list (instance trajectory_msgs::JointTrajectoryPoint
+                                :init
+                                :positions (mapcar #'deg2rad av-list)
+                                :time_from_start (ros::time (/ tm 1000)))))
+          (send goal :goal :trajectory traj)
+          (send _right-hand-action :send-goal goal)
+          (when wait
+            (send _right-hand-action :wait-for-result))
+          (send _right-hand-action :get-result)
```

Comment:
`:get-result` never returns nil.
Even if connection to actionlib was failed, it returns initial `control_msgs::followjointtrajectoryresult`